LanternExtractor/EQ/Wld/Exporters/GltfWriter.cs

```diff
@@ -1035,7 +1037,7 @@ private IMeshBuilder<MaterialBuilder> InstantiateMeshBuilder(string meshName, bo
                 {
                     var vertexIndex = gltfVertexPositionToWldVertexIndex[vertexGeometry];
                     var wldVertexPositionForFrame = vertexPositionsForFrame[vertexIndex];
-                    var newPosition = (wldVertexPositionForFrame + mesh.Center).ToVector3(true);
+                    var newPosition = Vector3.Transform((wldVertexPositionForFrame + mesh.Center).ToVector3(true), mirrorAxis ? MirrorXAxisMatrix : Matrix4x4.Identity);
```

Comment:
Something else outside the scope of this PR, but I should probably clean up the inconsistencies of how axes are flipped throughout this code. There are Vector3 extension methods with optional boolean arguments, and then sometimes we apply the Mirror matrix.

Reply:
Yes some universal helper utils with some docs around the method would help, I agree. I think the vec3 transform is the most straightforward personally.